org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/extensions/contentmodel/XMLSchemaTypeDefinitionExtensionsTest.java

```diff
@@ -145,36 +143,4 @@ public void localXSAttributeWithCatalog() throws BadLocationException, Malformed
 
 	}
 
-	/**
-	 * @see https://github.com/angelozerr/lsp4xml/issues/311
-	 * 
-	 * @throws BadLocationException
-	 */
-	@Test
-	public void issue311() throws BadLocationException {

```

Comment:
why is this test removed?

Reply:
Because it was an error of copy/paste from XMLSchemaCompletionExtensiontest. This test already exists in this class